plugins/arElasticSearchPlugin/lib/model/arElasticSearchModelBase.class.php

```diff
@@ -126,8 +126,8 @@ public static function serializeI18ns($id, array $classes, $options = array())
       }
     }
 
-    // Remove duplicated cultures
-    $i18ns['languages'] = array_unique($i18ns['languages']);
+    // Remove duplicated cultures from language values
+    $i18ns['languages'] = array_unique(array_values($i18ns['languages']));
```

Comment:
Is `array_values()` necessary here? `array_unique()` only compares array values anyway.

Reply:
There's a weird issue I ran into that necessitated this...

The TLDR is that not having that would result, in some cases, in data like this for languages:

`array(0 => 'en', 2 => 'fr')`

rather than

`array(0 => 'en', 1 => 'fr')`

And Elastica would index the former differently than the latter and an ES parse error would result.

Reply:
Ahh, okay.  Thanks for the explanation. :)

Reply:
That's my bad, thanks @mcantelon!